src/Codeception/Codecept.php

```diff
@@ -37,29 +37,33 @@ class Codecept
      * @var array
      */
     protected $options = [
-        'silent'        => false,
-        'debug'         => false,
-        'steps'         => false,
-        'html'          => false,
-        'xml'           => false,
-        'json'          => false,
-        'tap'           => false,
-        'report'        => false,
-        'colors'        => false,
-        'coverage'      => false,
-        'coverage-xml'  => false,
-        'coverage-html' => false,
-        'coverage-text' => false,
-        'groups'        => null,
-        'excludeGroups' => null,
-        'filter'        => null,
-        'env'           => null,
-        'fail-fast'     => false,
-        'ansi'          => true,
-        'verbosity'     => 1,
-        'interactive'   => true,
-        'no-rebuild'    => false,
-        'quiet'         => false,
+        'silent'                            => false,
+        'debug'                             => false,
+        'steps'                             => false,
+        'html'                              => false,
+        'xml'                               => false,
+        'logIncompleteSkipped'              => false,
+        'json'                              => false,
+        'tap'                               => false,
+        'report'                            => false,
+        'colors'                            => false,
+        'coverage'                          => false,
+        'coverage-xml'                      => false,
+        'coverage-html'                     => false,
+        'coverage-text'                     => false,
+        'groups'                            => null,
+        'excludeGroups'                     => null,
+        'filter'                            => null,
+        'env'                               => null,
+        'fail-fast'                         => false,
+        'ansi'                              => true,
+        'verbosity'                         => 1,
+        'interactive'                       => true,
+        'no-rebuild'                        => false,
+        'quiet'                             => false,
+        'reportUselessTests'                => false,
```

Comment:
this is not a configuration options but a runner options passed with parameters like `--fail-fast`, `--no-rebuild`. This is a config option so probably it should be set inside `Codeception\Configuration` class.

Yes, I know that might sound confusing :upside_down_face: